_posts/2020-11-16-october-rubygems-update.md

```diff
@@ -0,0 +1,56 @@
+
+## RubyGems News
+
+In October we released a new Bundler version --- **`2.2.0.rc.2`**. We also upgraded Bundler's & Rubygems' vendored Molinillo to **`0.7.0`**  (it's latest release). - [#3402](https://github.com/rubygems/rubygems/pull/3402), [#3388](https://github.com/rubygems/rubygems/pull/3388). 
+
+We also made the following changes and improvements:
+* fixed an [issue with the `specific_platform` setting.](https://github.com/rubygems/rubygems/pull/4022) 
+* merged a PR that stops changing the CWD for building extensions, which should allow concurrent extension compilation without any contention.
+* worked on PRs to improve independence between test and lib code (simplifies and helps packagers), and changes to improve specs.
+* added improvements to require more default gems lazily.
+* fixed an issue with help commands when Bundler installed from `ruby-core` installer and `man` are not available. - [#3997](https://github.com/rubygems/rubygems/pull/3997)
+* wrapped up a PR to allow installing plugins from local paths. - [#4020](https://github.com/rubygems/rubygems/pull/4020)
+* fixed daily Bundler CI by marking the new `pathname` default gem as unsupported. - [#4029](https://github.com/rubygems/rubygems/pull/4029)
+* fixed a `Kernel.warn` stackoverflow. - [#3987](https://github.com/rubygems/rubygems/pull/3987)
+* improved the deprecation path for gem query. - [#4021](https://github.com/rubygems/rubygems/pull/4021)
+* fixed issues with Bundler not loading RubyGems plugins in `$LOAD_PATH` locations, which was affecting some version managers. - [#3534](https://github.com/rubygems/rubygems/pull/3534)
+
+As always, we continue responding to RubyGems & Bundler issues and PRs, doing issue triage on both old and new issues, and reducing the number of open tickets.
+
+In October, Rubygems gained [143 new commits](https://github.com/rubygems/rubygems/compare/master@%7B2020-10-01%7D...master@%7B2020-10-31%7D), contributed by 11 authors. There were 2889 additions and 1705 deletions across 889 files.
+
+## RubyGems.org News
+
+This month we implemented a [bulk update to RubyGem downloads count](https://github.com/rubygems/rubygems.org/pull/2527) to reduce the processing time for `FastlyLogProcessor` by about 95%, up to 40 seconds. We studied an old HackerOne Report and proposed a solution for it that will get implemented into RubyGems.org. 
+
+We also made the following changes and improvements:
+
+* imported Fastly rubygems.org production configuration to Terraform. - [#4](https://github.com/rubygems/rubygems-terraform/pull/4) 
+* updated Fastly `vcl` to unset X-Forwarded-Host from requests to fix a HO report. - [#5](https://github.com/rubygems/rubygems-terraform/pull/5)
+* added regex whitelist for URL on honeycomb logs export to ensure we don't inadvertently send any sensitive information. - [#7](https://github.com/rubygems/rubygems-terraform/pull/7)
```

Comment:
I am not sure if we are adding any value by adding links to private repo here. I link https://github.com/rubygems/rubygems-terraform in my task updates as some people in the channel may have access.

Reply:
True. I'll take the links out just mentioning the task works as well.